src/benchmarks/micro/coreclr/perflab/CastingPerf.cs

```diff
@@ -114,6 +116,11 @@ public class CastingPerf
 
         public static Object[] myClass1Arr;
         public static Object[] myClass2Arr;
+        public static Object myObj;
+
+        // to make 
```

Comment:
Should be updated (https://github.com/dotnet/performance/pull/922/files#r332812461?)

Reply:
yes, thanks!

Reply:
Would `MethodImplOptions.NoOptimization` (maybe in conjunction with `NoInlining`) be even more safe?

Reply:
`NoOptimization` would have effect on the method body, which is here empty. 
We do not need to go too far with this. In theory an all-knowing perfect optimizer can see through tricks like this and would be hard to defeat. In practice non-inlinable call should be sufficient for the foreseeable future.